src/utilities/HYPRE_utilities.h

```diff
@@ -229,7 +229,9 @@ HYPRE_Int HYPRE_SetGPUMemoryPoolSize(HYPRE_Int bin_growth, HYPRE_Int min_bin, HY
  * HYPRE handle
  *--------------------------------------------------------------------------*/
 
-HYPRE_Int HYPRE_SetSpGemmUseCusparse( HYPRE_Int use_cusparse );
+/* Backwards compatibility with HYPRE_SetSpGemmUseCusparse() */
+#define HYPRE_SetSpGemmUseCusparse(use_vendor) HYPRE_SetSpGemmUseVendor(use_vendor)
```

Comment:
@liruipeng , is this what you had in mind for providing backward compatibility?

Reply:
yep. Thanks!